xml/admin_nfsganesha.xml

```diff
@@ -39,403 +28,148 @@
-     represented as NFS directories. Files and directories subordinate to
-     buckets are each represented as objects, following S3 prefix and delimiter
-     conventions, with <literal>/</literal> being the only supported path delimiter.
-   </para>
- </sect1>
-
- <sect1 xml:id="ceph-nfsganesha-config">
-  <title>Configuration</title>
-
-  <para>
-   For a list of all parameters available within the configuration file, see:
-  </para>
-
-  <itemizedlist>
+ <para>
+   Each &ganesha; service consists of a configuration hierarchy that contains:
+ </para>
+ <itemizedlist>
    <listitem>
-    <para>
-     <command>man ganesha-config</command>
-    </para>
+     <para>
+       A bootstrap configuration
+     </para>
    </listitem>
    <listitem>
-    <para>
-     <command>man ganesha-ceph-config</command> for &cephfs; File System
-     Abstraction Layer (FSAL) options.
-    </para>
+     <para>
+       A per-daemon or per-service configuration
+     </para>
    </listitem>
    <listitem>
-    <para>
-     <command>man ganesha-rgw-config</command> for &ogw; FSAL options.
-    </para>
+     <para>
+       An export configuration
+     </para>
    </listitem>
-  </itemizedlist>
+ </itemizedlist>
+ <para>
+   The bootstrap configuration is the minimal configuration to start the
+   <literal>nfs-ganesha</literal> daemon within a container. Each bootstrap
+   configuration will contain a <literal>%url</literal> directive that includes
+   any additional configuration from the &rados; common configuration object.
+   The common configuration object can include additional <literal>%url</literal>
+   directives for each of the NFS exports defined in the export &rados;
+   configuration objects.
+ </para>
 
+ <sect1 xml:id="ceph-nfsganesha-services">
+  <title>Starting or Restarting &ganesha;</title>
+  <para>
+   To start the &ganesha; service, run:
+  </para>
+ <screen>&prompt.cephuser;ceph orch start nfs.<replaceable>SERVICE_ID</replaceable></screen>
+  <para>
+   To restart the entire &ganesha; service, run:
+  </para>
+ <screen>&prompt.cephuser;ceph orch restart nfs.<replaceable>SERVICE_ID</replaceable></screen>
+  <para>
+    If you only want to restart a single &ganesha; daemon, run:
+  </para>
+ <screen>&prompt.cephuser;ceph orch daemon restart nfs.<replaceable>SERVICE_ID</replaceable></screen>
   <para>
-   This section includes information to help you configure the &ganesha; server
-   to export the cluster data accessible via &ogw; and &cephfs;.
+   When &ganesha; is started or restarted, it has a grace timeout of 90 seconds
+   for NFS v4. During the grace period, new requests from clients are actively
+   rejected. Hence, clients may face a slowdown of requests when NFS is in
+   grace state.
   </para>
+ </sect1>
 
-  <important>
-   <title>Restart the &ganesha; Service</title>
+ <sect1 xml:id="ceph-nfsganesha-createcephfs">
+   <title>Create a &cephfs; Volume</title>
    <para>
```

Comment:
Remove this section? An NFS export can be created against any existing cephfs or RGW .. maybe the cephfs filesystem creation belongs in different chapter?

Reply:
That was in your docs, was it for another reason? https://confluence.suse.com/pages/viewpage.action?spaceKey=SUSEEnterpriseStorage&title=nfs-ganesha#nfs-ganesha-CreateaCephfsVolume

Reply:
The doc was intended as a rough shorthand guide for our test team. I think removing it from the NFS section makes sense, because all the nuances of cephfs / RGW probably should be documented elsewhere ...

Reply:
It's been removed :)